runtime/mainnet/src/lib.rs

```diff
@@ -378,13 +383,41 @@ parameter_types! {
 	/// Relay Chain `TransactionByteFee` / 10
 	pub const TransactionByteFee: Balance = fee::TRANSACTION_BYTE_FEE;
 	pub SudoAddress: AccountId = AccountId::from_ss58check("15NMV2JX1NeMwarQiiZvuJ8ixUcvayFDcu1F9Wz1HNpSc8gP").expect("sudo address is valid SS58");
+	pub MaintenanceAccount: AccountId = AccountId::from_ss58check("1Y3M8pnn3rJcxQn46SbocHcUHYfs4j8W2zHX7XNK99LGSVe").expect("maintenance address is valid SS58");
 }
 
+/// DealWithFees is used to handle fees and tips in the OnChargeTransaction trait,
+/// by implementing OnUnbalanced.
+pub struct DealWithFees;
```

Comment:
Not to scope creep, and this should probably be dealt with in the future rather than now, but consider that we will have multiple runtimes, even if just devnet and mainnet.

A refactor to take a generic type which is something like Get<Map<AccountId, Percent>> means we can move this functionality into common, with tests. Each runtime then just uses it to configure via a type defined via parameter_types which defines the split. It is then more explicit, rather than buried in some struct impl.

Could be over engineering, but maintaining multiple runtimes has already been cited as a cause for headache. A minor refactor as suggested can reduce this and improve the overall codebase.

Reply:
CC @Daanvdplas
https://github.com/r0gue-io/pop-node/pull/428#discussion_r1928753488

If we need it lets put it in. 
If you want it generic then will but im going round in circles now.

Reply:
Wasn't aware of that, no further action required.

Note that I only make suggestions, no hard requirements.